sdks/typescript/package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "apache-beam",
-  "version": "2.54.0-SNAPSHOT",
+  "version": "2.57.0-SNAPSHOT",
```

Comment:
Though this is an unrelated change -- should this version bump be automated as part of cut-release-branch script? cc: @robertwb

Reply:
I'll remove it, but I didn't change it myself... won't it just get bumped again automatically?

Reply:
Odd, I pushed a commit but it's not showing directly in the PR's file changes. It's reflected properly in my branch and in the commit's changes, though.